acronis-db-bench/db.go

```diff
@@ -128,15 +128,18 @@ func NewDBConnector(dbOpts *DatabaseOpts, workerID int, logger *benchmark.Logger
 
 	var queryLogger, readedRowsLogger, queryTimeLogger db.Logger
 	if dbOpts.LogQueries {
-		queryLogger = &dbLogger{level: benchmark.LogTrace, worker: workerID, logger: logger}
+		logger.LogLevel = benchmark.LogInfo
+		queryLogger = &dbLogger{level: benchmark.LogInfo, worker: workerID, logger: logger}
```

Comment:
why LogTrace changed to LogInfo ?

Reply:
It is because queries are logged only when severity is at least LogInfo